Yoast/javascript
# Stem and create forms of very irregular German verbs

## Summary
* [Non-user-facing] Improves keyphrase recognition for additional irregular verbs in German (e.g., `brennen`, `senden`, `kennen`).
* [Non-user-facing] Adds functionality to properly stem and create forms of very irregular verbs.

## Test instructions
This PR can be tested by following these steps:
User-facing:
* Set your site to German and make a new post with 150+ words.
* In the keyphrase, add a form of one of the following verbs: `brennen`, `bringen`,
`kennen`, `nennen`, `rennen`, `senden`, `wenden`, `werden`.
* In the text, add other forms of the same word. You can use a dictionary to find forms, e.g. https://de.wiktionary.org/wiki/Flexion:nennen.

Non-user-facing:
* Make sure the correct word forms are created for the following verbs (via unit tests): _sein_, _haben_, _tun_
* Make sure each form of the above verbs stems to its infinitive form (so _sein_, _haben_, and _tun_) (via unit tests).

Fixes #163
